vertx-pojo-mapper-common-test/src/main/java/de/braintags/io/vertx/pojomapper/testdatastore/DatastoreBaseTest.java

```diff
@@ -44,10 +44,10 @@
 /**
  * Base test to init an IDataStore for working with an {@link IDataStore}. See method {@link #getDataStore(TestContext)}
  * on how to init
- * 
- * 
+ *
```

Comment:
I accidentally had "remove whitespace at end of lines" enabled as save format action. Disabled it.